bindings/cs/MongoDB.Libmongocrypt.Test/BasicTests.cs

```diff
@@ -260,6 +260,50 @@ public void EncryptExplicitStepwise()
             }
         }
 
+        [Fact]
+        public void TestAwsKeyCreationWithEndPoint()
+        {
+            var endpoint = "kms.us-east-1.amazonaws.com";
+            var keyId = new AwsKeyId(
+                customerMasterKey: "cmk",
+                region: "us-east-1",
+                endpoint: endpoint);
+            var key = new AwsKmsCredentials(awsSecretAccessKey: "us-east-1", awsAccessKeyId: "us-east-1");
+
+            using (var cryptClient = CryptClientFactory.Create(new CryptOptions(CreateCredentialsMap(key))))
+            using (var context = cryptClient.StartCreateDataKeyContext(keyId))
+            {
+                var (_, dataKeyDocument) = ProcessContextToCompletion(context, isKmsDecrypt: false);
+                dataKeyDocument.Should().NotBeNull();
```

Comment:
Vincent,  `dataKeyDocument` doesn't contain the `endpoint` value. Do we expect this value here?

Reply:
Spoke with Kevin, there may be a bug in libmongocrypt

Reply:
Let's change the test to expect the endpoint value in anticipation of the new version of libmongocrypt :)